.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -74,6 +74,7 @@
     <inspection_tool class="BigDecimalEquals" enabled="true" level="WARNING" enabled_by_default="true" />
     <inspection_tool class="BreakStatement" enabled="true" level="WARNING" enabled_by_default="true" />
     <inspection_tool class="BreakStatementWithLabel" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="BusyWait" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
Please synchronize this file with the `config` module. It might be that your local settings are infiltrating the repository.

Reply:
I actually got these changes from the `config/pull`. Will check it and probably revert it.